Kudu.Contracts/Functions/FunctionEnvelope.cs

```diff
@@ -1,41 +1,41 @@
 ﻿using Kudu.Contracts.Infrastructure;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 using System;
 using System.Collections.Generic;
+using System.Text.Json.Nodes;
+using System.Text.Json.Serialization;
 
 namespace Kudu.Core.Functions
 {
     public class FunctionEnvelope : INamedObject
     {
-        [JsonProperty(PropertyName = "name")]
+        [JsonPropertyName("name")]
         public string Name { get; set; }
 
-        [JsonProperty(PropertyName = "function_app_id")]
+        [JsonPropertyName("function_app_id")]
         public string FunctionAppId { get; set; }
 
-        [JsonProperty(PropertyName = "script_root_path_href")]
+        [JsonPropertyName("script_root_path_href")]
         public Uri ScriptRootPathHref { get; set; }
 
-        [JsonProperty(PropertyName = "script_href")]
+        [JsonPropertyName("script_href")]
         public Uri ScriptHref { get; set; }
 
-        [JsonProperty(PropertyName = "config_href")]
+        [JsonPropertyName("config_href")]
         public Uri ConfigHref { get; set; }
 
-        [JsonProperty(PropertyName = "secrets_file_href")]
+        [JsonPropertyName("secrets_file_href")]
         public Uri SecretsFileHref { get; set; }
 
-        [JsonProperty(PropertyName = "href")]
+        [JsonPropertyName("href")]
         public Uri Href { get; set; }
 
-        [JsonProperty(PropertyName = "config")]
-        public JObject Config { get; set; }
```

Comment:
Make sure functional are passing and we are not breaking existing functionality.